### ISSUE TITLE
Limits on settings values are unfriendly to hidpi displays

### ISSUE THREAD

**Issue by fastcat** [hedayaty/NetSpeed#38]
Some of the limits on element sizes for the UI don't allow high enough values to avoid elipsizing on hidpi displays.

Ideally the settings values should be hidpi aware so that the defaults are reasonable there :)

**Comment by friederbluemle** [comment]
Yes, with the default settings barely anything is visible on a UHD screen. I first had to manually adjust the values. Additionally, the spacing is off. A fix would be great. Thanks

**Comment by hedayaty** [comment]
Any suggestion for range/default values?

**Comment by JoranDox** [comment]
For me, setting the label sizes to 100 is barely enough, allowing this to go to 1000 or even unlimited would make perfect sense to me, but I don't know about the internals and whether this is possible.

**Comment by hedayaty** [comment]
Added HiDPI settings

**Comment by aunetx** [comment]
Would you like me to try to implement automatic hidpi detection?
With this setting, the behaviour is inconsistent between xorg and wayland sessions, the UI being too big under wayland (but it works then with HIDPI factor=1).

If you want (and it happens to other people), I can implement a way to check whether it is running wayland or xorg, and to then always use the best default for each session.

**Comment by hedayaty** [comment]
At the point I do not use HIDPI but I think more and more people are using them these days.
@aunetx I would appropriate if you can add something.
